torch_xla/csrc/helpers.cpp

```diff
@@ -394,4 +397,10 @@ xla::XlaOp XlaHelpers::PromotedBinaryOp(
   return bin_op(vops.first, vops.second);
 }
 
+void XlaHelpers::CheckDuplicateInDimList(std::vector<xla::int64> dims) {
```

Comment:
This function has one use. Should be probably just inlined at call site.
The implementation is a bit strange. Requiring a copy of the vector just to do a sort.